<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-47</y>
       <width>301</width>
       <height>183</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 1</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_A">
        <property name="text">
         <string>“Коефіцієнт А:”</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_A"/>
      </item>
      <item>
       <widget class="QLabel" name="label_B">
        <property name="text">
         <string>	“Коефіцієнт B:”</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_B"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calc">
        <property name="text">
         <string>Обчилити</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_result">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget_fileContent">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>133</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 2</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QQuickWidget" name="quickWidget">
          <property name="resizeMode">
           <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_history"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu_2">
    <property name="title">
     <string>Вихід</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="Menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Відкрити</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Вихід з програми</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
